apps/api/src/auth/guard/auth/auth.guard.ts

```diff
@@ -12,6 +12,7 @@ import { PrismaService } from '../../../prisma/prisma.service'
 import { ONBOARDING_BYPASSED } from '../../../decorators/bypass-onboarding.decorator'
 import { AuthenticatedUserContext } from '../../auth.types'
 import { toSHA256 } from '../../../common/to-sha256'
+import { EnvSchema } from 'src/common/env/env.schema'
```

Comment:
![image](https://github.com/keyshade-xyz/keyshade/assets/83924254/156417ec-e2ef-4caa-8e1c-f270a57a9a1a)

Remove `src` from the import or it will break. Use a relative path.